lib/account/account-manager.js

```diff
@@ -46,24 +104,28 @@
  * @property {string} regId Unhashed registration id sent to client
  */
 
-module.exports = function (accountService, config, hotp, otDate, promise, random) {
+module.exports = function (accountService, base64, config, hotp, otDate, promise, random, secureHash) {
+    var accountConfig;
+
+    accountConfig = config.account || {};
+
     /**
      * Checks to see if the values for MFA are valid and will
      * throw if they are not.
      *
      * @param {string} mfaKey
-     * @param {string} previousMfa
      * @param {string} currentMfa
+     * @param {string} [previousMfa]
```

Comment:
When you make this parameter optional, how are you guaranteeing that both the current and previous have been sent during signup?